definitions/test/discovery_test_HuisartsVVTInzage.json

```diff
@@ -0,0 +1,127 @@
+{
+  "id": "test:HuisartsVVTInzage",
+  "endpoint": "https://nuts.test-datakluis.shared.topicuszorg.nl/discovery/test:HuisartsVVTInzage",
+  "presentation_max_validity": 259200,
+  "presentation_definition": {
+    "id": "test:HuisartsVVTInzage",
+    "format": {
+      "ldp_vc": {
+        "proof_type": [
+          "JsonWebSignature2020"
+        ]
+      },
+      "jwt_vc": {
+        "alg": [
+          "PS256"
+        ]
+      },
+      "ldp_vp": {
+        "proof_type": [
+          "JsonWebSignature2020"
+        ]
+      },
+      "jwt_vp": {
+        "alg": [
+          "ES256"
+        ]
+      }
+    },
+    "input_descriptors": [
+      {
+        "id": "id_uzicert_uracredential",
+        "name": "Care organization identity from CIBG-issued UZI-server certificate",
+        "purpose": "Finding a care organization for authorizing access to medical metadata.",
+        "constraints": {
+          "fields": [
+            {
+              "path": [
+                "$.vc.type",

```

Comment:
Mapping currently starts within the `.vc`, so this will (currently) never have a match
```suggestion
```